<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startModelingButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Старт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopModelingButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Стоп</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearParametersButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Сброс</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="firstCashierMeanTimeSpin">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="secondCashierMeanTimeSpin">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="firstCashierLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Среднее время 1 кассира</string>
    </property>
   </widget>
   <widget class="QLabel" name="secondCashierLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Среднее время 2 кассира</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setFirstCashierButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Установить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setSecondCashierButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Установить</string>
    </property>
   </widget>
   <widget class="QLabel" name="meanTimeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Среднее время прихода 
 посетителя</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="meanCustomerTime">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="setCustomerMeanButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Установить</string>
    </property>
   </widget>
   <widget class="QLabel" name="addressLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Адрес сервера
 В формате (host:port)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="serverAddressEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="connectServerButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Соединиться</string>
    </property>
   </widget>
   <widget class="QLabel" name="configurationLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Конфигурация</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>20</width>
      <height>191</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="statsLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>5</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Статистика</string>
    </property>
   </widget>
   <widget class="QPushButton" name="statsUpdateButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>270</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить</string>
    </property>
   </widget>
   <widget class="QLabel" name="firstMeanLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Среднее 1 кассира</string>
    </property>
   </widget>
   <widget class="QLabel" name="firstSTDLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>СКО 2 кассира</string>
    </property>
   </widget>
   <widget class="QLabel" name="secondMeanLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Среднее 1 кассира</string>
    </property>
   </widget>
   <widget class="QLabel" name="secondSTDLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>СКО 2 кассира</string>
    </property>
   </widget>
   <widget class="QLabel" name="firstMeanValue">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="firstSTDValue">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="secondMeanValue">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>120</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="secondSTDValue">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>170</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0.00</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>734</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Моделирование касс</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
